ztp/source-crs/SriovFecSubscriptionNS.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: vran-acceleration-operators
+  annotations:
+    workload.openshift.io/allowed: management
```

Comment:
This is not part of the mgmt workload.

Reply:
This will be used for FPGAs on our favorite customer laboratory, is another requirements from him.